examples/minikube/README.md

```diff
@@ -9,6 +9,7 @@ You might want to review those settings, especially if you are lucky enough to h
 
 * kubectl apply -f namespace.yml
 * kubectl apply -f config.yml
+* kubectl apply -f rbac.yml
```

Comment:
Does it fail if you don't have RBAC enabled in your minikube ?

Reply:
Yeah, complains that it can't find the service account.